app/src/main/res/values/strings.xml

```diff
@@ -88,4 +88,9 @@
     <string name="msg_get_user_info_failed"> .Please try to login again!</string>
     <string name="description_back_to_timeline">Press to back to timeline</string>
     <string name="description_vote_for_post">Vote for this post</string>
+    <string name="msg_error_init">Sorry, you can not init this audio. Please try again later!</string>
+    <string name="msg_error_start">Sorry, an error has occurred. Please try again later!</string>
+    <string name="description_play_audio">Play the detail audio</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.description_play_audio` appears to be unused
 Unused resources make applications larger and slow down builds.